torch/_dynamo/variables/nn_module.py

```diff
@@ -193,8 +193,9 @@ def call_function(
 
         @contextmanager
         def record_nn_module_stack():
+            fully_qualified_name = self.source.name()
             try:
-                tx.nn_module_stack[self.module_key] = type(mod)
+                tx.nn_module_stack[self.module_key] = (fully_qualified_name, type(mod))
```

Comment:
BTW, can you update the type annotation?
https://github.com/pytorch/pytorch/blob/a17a7ccc92144452e0fe51e02f21f1f1ba88118a/torch/_dynamo/symbolic_convert.py#L1621

Reply:
Sure!